gwvolman/lib/dataone/metadata.py

```diff
@@ -261,17 +265,16 @@ def create_eml_doc(self, eml_pid, manifest, user_id, manifest_size,
         dataset = ET.SubElement(ns, 'dataset')
         ET.SubElement(dataset, 'title').text = manifest['schema:name']
 
-        first_name = manifest['createdBy']['schema:givenName']
-        last_name = manifest['createdBy']['schema:familyName']
-        email = manifest['createdBy']['schema:email']
 
         """
-        Create a `creator` section, using the information in the
-         `model.user` object to provide values.
+        Create a `creator` section for each Tale author.
         """
-        creator = ET.SubElement(dataset, 'creator')
-        self.set_user_name(creator, first_name, last_name)
-        self.set_user_contact(creator, user_id, email)
+        for author in manifest['schema:author']:
+            creator_attribute = ET.SubElement(dataset, 'creator')
+            first_name = author['schema:givenName']
+            last_name = author['schema:familyName']
+            contact_info = author['@id']
```

Comment:
Same comment as above, to match EML, `contact_info` should be `user_id` or similar.